plugins/masters-tournament/manager.py

```
@@ -133,6 +133,12 @@ def __init__(self, plugin_id, config, display_manager, cache_manager, plugin_man
         self._last_player_card_advance = 0.0
         self._player_card_interval = config.get("player_card_duration", 8)
 
+        # Vegas scroll mode: fixed card block width. Cards render at
+        # (scroll_card_width × display_height) regardless of the panel width
```

Ruff (0.15.9)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)